### DISCUSSION TITLE
[BUG] zwift sur android ne voit pas qdomyos

### DISCUSSION THREAD

**Discussion by Strangistra** [cagnulein/qdomyos-zwift#247] [Q&A]
**Describe the BUG**
Bonjour
Je viens de faire l'acquisition d'un vélo domyos e-bike 500 avec connections Bluetooth. J'ai acheté Qdomyos/zwift et l'ai installé sur mon téléphone portable. La connexion entre le vélo et l'application Qdomyos est OK, mais lorsque j'ouvre zwift l'application ne trouve aucun périphérique à connecter.

**Expected behavior**
Je veux pouvoir connecter mon vélo sur l'application zwift.

**Screenshots**

**Qdomyos (please complete the following information):**
 - OS: Android
 - Version : 2.4.24

**Smartphone (please complete the following information):**
 - Device: Redmi9
 - OS: android
 - Version : 10QP1A.190711.020

**Additional context**
J'ai l'ensemble des applications sur mon smartphone. Je ne passe pas par un ordinateur pour streamer sur la TV. C'est mon téléphone qui le fait.

**Comment by cagnulein** [comment]
You need a device for qdomyos and one for zwift. Are you aware of this?

Il sab 3 apr 2021, 21:00 Strangistra ***@***.***> ha scritto:

**Comment by Strangistra** [comment]
Non je pensais que l'applications Qdomyos faisait la relation au sein même du téléphone. Donc si je comprends bien il faut absolument que zwift soit sur PC Portable? 
Qdomyos qui est sur mon telephone fait la relation avec le vélo Domyos 500 en Bluetooth et par la suite mon téléphone fait la relation avec mon ordinateur via aussi le Bluetooth?  C'est bien ça.

**Comment by cagnulein** [comment]
Yes, you can use zwift on a pc or on another android or iOS device :)


Il sab 3 apr 2021, 21:10 Strangistra ***@***.***> ha scritto:
